go.mod

```diff
@@ -162,9 +162,9 @@ require (
 	go.starlark.net v0.0.0-20230525235612-a134d8f9ddca // indirect
 	go.uber.org/multierr v1.11.0 // indirect
 	golang.org/x/exp v0.0.0-20240213143201-ec583247a57a // indirect
-	golang.org/x/oauth2 v0.17.0 // indirect
-	golang.org/x/sys v0.22.0 // indirect
-	golang.org/x/term v0.22.0 // indirect
+	golang.org/x/oauth2 v0.18.0 // indirect
+	golang.org/x/sys v0.21.0 // indirect
+	golang.org/x/term v0.21.0 // indirect
```

Comment:
No idea, I didn't notice.. I just ran go mod tidy.. Will reverse it

Reply:
oh, never mind. Its mostly because I've changed the everest-operator package to EVEREST-1247 to be able to use the GetNextUpgrade function. It will be reverted back once I merged the everest-operator PR and clean-up go mod finally..